build.sbt

```diff
@@ -2,10 +2,10 @@ organization := "com.spotify.data"
 name := "gcs-tools"
 version := "0.1.8-SNAPSHOT"
 
-val gcsVersion = "1.6.2-hadoop2"
+val gcsVersion = "1.6.3-hadoop2"
 val hadoopVersion = "2.7.4"
-val avroVersion = "1.8.2"
-val parquetVersion = "1.9.0"
+val avroVersion = "1.9.0"
+val parquetVersion = "1.10.1"
```

Comment:
we can even possibly bump this to 1.11.0

Reply:
sure.

Reply:
for some reason, sbt is not able to find 1.11.0